src/atoms/Flex/Flex.native.js

```diff
@@ -19,7 +19,12 @@ const Flex = styled(
     children,
     style,
     ...props
-  }) => React.cloneElement(children, { ...props, style: [style, children.props.style] }),
+  }) => {
+    return React.cloneElement(children, {
+      ...props,
+      style: [...style, children.props.style].filter(Boolean),
+    });
+  },
 )`
```

Comment:
Can't we just do this?

```suggestion
  }) => React.cloneElement(children, { ...props, style: [style, children.props.style].filter(Boolean) }),
```